templates/rhdm70-kieserver.yaml

```diff
@@ -0,0 +1,347 @@
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    description: Application template for Red Hat Decision Manager Execution Server 7.0 application.
+    iconClass: icon-jboss
+    tags: rhdm,jboss,xpaas
+    version: 1.4.0
+    openshift.io/display-name: Red Hat Decision Manager Execution Server 7.0 (Ephemeral with https)
+  name: rhdm70-kieserver
+labels:
+  template: rhdm70-kieserver
+  xpaas: 1.4.0
+message: A new Decision Manager Execution Server application has been created in your
+  project. Please be sure to create the "rhdm-service-account" service account and
+  the secret named "${HTTPS_SECRET}" containing the ${HTTPS_KEYSTORE} file used for
+  serving secure content.
+parameters:
+- displayName: Application Name
+  description: The name for the application.
+  name: APPLICATION_NAME
+  value: myapp
+  required: true
+- displayName: EAP Admin User
+  description: EAP administrator username
+  name: ADMIN_USERNAME
+  value: eapadmin
+  required: false
+- displayName: EAP Admin Password
+  description: EAP administrator password
+  name: ADMIN_PASSWORD
+  from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: false
+- displayName: KIE Admin User
+  description: KIE administrator username
+  name: KIE_ADMIN_USER
+  value: adminUser
+  required: false
+- displayName: KIE Admin Password
+  description: KIE administrator password
+  name: KIE_ADMIN_PWD
+  from: "[a-zA-Z]{6}[0-9]{1}!"
+  generate: expression
+  required: false
+- displayName: KIE Server Container Deployment
+  description: 'KIE Server Container deployment configuration in format: containerId=groupId:artifactId:version|c2=g2:a2:v2'
+  name: KIE_SERVER_CONTAINER_DEPLOYMENT
```

Comment:
Is this template intend to be used by the decision manager right?
If so I believe it is not needed.